<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>1</number>
     </property>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="input_button">
       <property name="text">
        <string>信息录入</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="date_edit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="num_edit">
       <property name="placeholderText">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="gender_combobox">
       <property name="placeholderText">
        <string>性别：</string>
       </property>
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit">
       <property name="placeholderText">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="class_combobox">
       <property name="placeholderText">
        <string>班级</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="num_edit2">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>学号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_course_button">
         <property name="text">
          <string>查询课程</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="score_edit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>成绩：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="score_input_button">
       <property name="text">
        <string>成绩录入</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="course_combobox">
       <property name="placeholderText">
        <string>课程ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
